src/components/dataset/query/QueryView.jsx

```diff
@@ -47,9 +47,11 @@ export class QueryView extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object,
     dataset: PropTypes.object,
-    datasetPolicies: PropTypes.object,
+    datasetPolicies: PropTypes.array,
```

Comment:
object to array?

Reply:
this was wrong the first time, I was getting console errors that it's actually an array so I changed the type